tests/providers/microsoft/azure/transfers/test_local_to_adls.py

```diff
@@ -28,11 +27,9 @@
 )
 
 TASK_ID = "test-adls-upload-operator"
-FILE_SYSTEM_NAME = "Fabric"
 LOCAL_PATH = "test/*"
 BAD_LOCAL_PATH = "test/**"
 REMOTE_PATH = "TEST-DIR"
-DATA = json.dumps({"name": "David", "surname": "Blain", "gender": "M"}).encode("utf-8")
```

Comment:
why?

Reply:
Same reason, test moved from location, this line was added to test the new operator and is not used by the operator being tested here, it is now in the test file (tests/providers/microsoft/azure/operators/test_adls_create.py) of the new operator:

Reply:
It is not.

https://github.com/apache/airflow/blob/354e633ddd7da5fb8310ef6bd1c5b9e5e8763848/tests/providers/microsoft/azure/transfers/test_local_to_adls.py#L35

this line exist in main. In this PR you are removing it.